packages/taco/integration-test/encrypt-decrypt.test.ts

```diff
@@ -128,5 +131,66 @@ describe.skipIf(!process.env.RUNNING_IN_CI)(
       // Verify decryption
       expect(decryptedMessageString).toEqual(messageString);
     }, 15000); // 15s timeout
+
+    // TODO: Unskip this test when wallet allowlist condition is implemented and running at the nodes as per https://github.com/nucypher/nucypher/issues/3600
+    test.skip('should encrypt and decrypt according to wallet allowlist condition', async () => {
```

Comment:
👍  - I assume the goal is the unskip this test before merging this PR.

Reply:
Yes, and the comment above should be removed when this is ready at the node side.